manifests/istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -45,20 +51,26 @@ template: |
   {{- end }}
     securityContext:
       allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
+      privileged: {{ .Values.global.proxy.privileged }}
       capabilities:
+    {{- if not .Values.istio_cni.enabled }}
         add:
         - NET_ADMIN
         - NET_RAW
+    {{- end }}
         drop:
         - ALL
-      privileged: {{ .Values.global.proxy.privileged }}
       readOnlyRootFilesystem: false
+    {{- if not .Values.istio_cni.enabled }}
       runAsGroup: 0
       runAsNonRoot: false
       runAsUser: 0
+    {{- else }}
+      runAsGroup: 1337
+      runAsUser: 1337
+      runAsNonRoot: true
+    {{- end }}
     restartPolicy: Always
-  {{- end }}
-  {{  end -}}
```

Comment:
You shouldn't remove this line. This doesn't match: it removes 2 `if`s and 3 `end`s. The changes in `install/kubernetes/helm/istio/files/injection-template.yaml` are correct.

Reply:
Great catch!
Is the template engine too robust or this is not used at all?  This should break any flow if the template is used.
Anyway I am update this file. Thank you so much!

Reply:
Maybe this means that we don't have any tests that run applications injected with Istio using the new installer?

Reply:
Talked with @howardjohn 
This one is for standalone injector which we are deprecating. I admit we don't have full coverage of the combination with all injectors X cni-on-off

Reply:
This syntax error should have triggered an error if *any* pod was injected in a test with that template, regardless of whether CNI is enabled.